lib/index.ts

```diff
@@ -117,7 +110,7 @@ function load(root, options) {
         mergePolicies(root, options).then(function (res) {
           if (debug.enabled) {
             debug('final policy:');
-            debug(JSON.stringify(res, '', 2));
+            debug(JSON.stringify(res, [], 2));
```

Comment:
The signature of `JSON.stringify` is like `JSON.stringify(value, replacer, space)`.
I think we need t replace the `''` with null since if we leave the `''` the output is converted to:

```node
const person = {
  firstName: 'John',
  age: 30,
  address: {
    state: 'CA',
    zip: '12345'
  }
}; 
JSON.stringify(person, [], 2)
"{}" 

// vs

JSON.stringify(person, null, 2)
'{
  "firstName": "John",
  "age": 30,
  "address": {
    "state": "CA",
    "zip": "12345"
  }
}' 

```

Reply:
Good catch! Just tested and `JSON.stringify(res, null, 2)` does give the same result as `JSON.stringify(res, '', 2)` 🙏

Reply:
I forgot to point out that: 

>Replacer can be either a function or an array.
>A function that alters the behaviour of the stringification process, or an array of strings and numbers that specifies properties of value to be included in the output.

[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify)

Reply:
Is `''` a function or an array?!!! :p